src/pages/EnablePayments/TermsPage/LongTermsForm.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import {View, Linking} from 'react-native';
+import styles from '../../../styles/styles';
+import Text from '../../../components/Text';
+import CollapsibleSection from '../../../components/CollapsibleSection';
+import {translateLocal} from '../../../libs/translate';
+import CONST from '../../../CONST';
+import Icon from '../../../components/Icon';
+import {Printer} from '../../../components/Icon/Expensicons';
+
+const termsData = [
+    {
+        title: translateLocal('termsStep.longTermsForm.openingAccountTitle'),
+        rightText: translateLocal('termsStep.feeAmountZero'),
+        details: translateLocal('termsStep.longTermsForm.openingAccountDetails'),
+    },
+    {
+        title: translateLocal('termsStep.monthlyFee'),
+        rightText: translateLocal('termsStep.feeAmountZero'),
+        details: translateLocal('termsStep.longTermsForm.monthlyFeeDetails'),
+    },
+    {
+        title: translateLocal('termsStep.longTermsForm.customerServiceTitle'),
+        subTitle: translateLocal('termsStep.longTermsForm.automated'),
+        rightText: translateLocal('termsStep.feeAmountZero'),
+        details: translateLocal('termsStep.longTermsForm.customerServiceDetails'),
+    },
+    {
+        title: translateLocal('termsStep.longTermsForm.customerServiceTitle'),
+        subTitle: translateLocal('termsStep.longTermsForm.liveAgent'),
+        rightText: translateLocal('termsStep.feeAmountZero'),
+        details: translateLocal('termsStep.longTermsForm.customerServiceDetails'),
+    },
+    {
+        title: translateLocal('termsStep.inactivity'),
+        rightText: translateLocal('termsStep.feeAmountZero'),
+        details: translateLocal('termsStep.longTermsForm.inactivityDetails'),
+    },
+    {
+        title: translateLocal('termsStep.longTermsForm.sendingFundsTitle'),
+        rightText: translateLocal('termsStep.feeAmountZero'),
+        details: translateLocal('termsStep.longTermsForm.sendingFundsDetails'),
+    },
+    {
+        title: translateLocal('termsStep.electronicFundsWithdrawal'),
+        subTitle: translateLocal('termsStep.standard'),
+        rightText: translateLocal('termsStep.feeAmountZero'),
+        details: translateLocal('termsStep.longTermsForm.electronicFundsStandardDetails'),
+    },
+    {
+        title: translateLocal('termsStep.electronicFundsWithdrawal'),
+        subTitle: translateLocal('termsStep.instant'),
+        rightText: translateLocal('termsStep.electronicFundsInstantFee'),
+        subRightText: translateLocal('termsStep.electronicFundsInstantFeeMin'),
+        details: translateLocal('termsStep.longTermsForm.electronicFundsStandardDetails'),
+    },
+];
+
+const getLongTermsSections = () => termsData.map((section, index) => (
+    // eslint-disable-next-line react/no-array-index-key
+    <View key={section.title + index}>
```

Comment:
They shouldn't be that long, and I don't think this will change after this. There is one duplicate line for `Customer Service` which is why I added the index (at first I added subTitle but that was also duplicated). I'm open to other ways of simplifying that though.

Reply:
Meh i think it's fine.